core/src/main/java/org/infinispan/util/CoreBlockHoundIntegration.java

```diff
@@ -29,6 +30,8 @@ public void applyTo(BlockHound.Builder builder) {
       // Block designates methods that should only hold a lock very briefly
       {
          CommonsBlockHoundIntegration.allowPublicMethodsToBlock(builder, OffHeapConcurrentMap.class);
+         // This acquires the locks from the above OffHeapConcurrentMap instance
```

Comment:
Not sure what `above` means here, but the `lruLock` is in `SegmentedBoundedOffHeapDataContainer` itself.

Reply:
The issue that identified it wasn't actually `lruLock`, but that it found it was the stamped lock from the off heap concurrent map.  However this fixes both :) I can change the comment.